tests/test_pipx_metadata_file.py

```diff
@@ -63,14 +63,17 @@
 
 
 def test_pipx_metadata_file_create(tmp_path):
-    pipx_metadata = PipxMetadata(tmp_path)
+    venv_dir = tmp_path / TEST_PACKAGE1.package
+    venv_dir.mkdir()
```

Comment:
Why is this necessary?

Reply:
This is required because of https://github.com/pipxproject/pipx/pull/491/files#diff-349c0b5cf5479f95786c8426e2656d2bR105-R109